docs/ManualDeploymentGuideScalarDLOnEKS.md

```diff
@@ -1,74 +1,53 @@
 
-## Step 1. Subscribe to Scalar DL Ledger in AWS Marketplace
+You can get the ScalarDL Ledger container image from [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3jdwfmqonx7a2). First, you need to subscribe to it. For more details on how to subscribe to ScalarDL Ledger in AWS Marketplace, see [Subscribe to Scalar products from AWS Marketplace - How to install Scalar products through AWS Marketplace](./AwsMarketplaceGuide.md#subscribe-to-scalar-products-from-aws-marketplace).
 
-You can get the Scalar DL Ledger container image from [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3jdwfmqonx7a2). First, you need to subscribe to it. Please refer to the following document to subscribe to Scalar DL Ledger in AWS Marketplace.  
+## Step 2. Create an EKS cluster
 
-* [How to install Scalar products through AWS Marketplace](./AwsMarketplaceGuide.md)
+Create an EKS cluster for the deployment of ScalarDL Ledger. See [Create an Amazon EKS cluster for Scalar products](./CreateEKSClusterForScalarProducts.md) for more details.
 
-Note: Please see the **Subscribe to Scalar products from AWS Marketplace** section in the above document.  
+## Step 3. Set up a database for ScalarDL Ledger
 
-## Step 2. Set up a database for Scalar DL Ledger
-
-Scalar DL Ledger uses Scalar DB in its internal to access a database and Scalar DB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy Scalar DL Ledger.  
-
-Please refer to the following document for more details.  
-
-* [Set up a database for Scalar DB/Scalar DL deployment on AWS](./SetupDatabaseForAWS.md)
-
-## Step 3. Create an EKS cluster
-
-Create an EKS cluster for the deployment of Scalar DL Ledger. Please refer to the following document for more details.  
-
-* [Create an EKS cluster for Scalar Products]() // TODO: Create a new document
+ScalarDL Ledger uses ScalarDB in its internal to access a database and ScalarDB supports [several databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md). You need to prepare a database before you deploy ScalarDL Ledger. See [Set up a database for ScalarDB/ScalarDL deployment on AWS](./SetupDatabaseForAWS.md) for more details.
 
 ## Step 4. Create a bastion server
 
-For executing some tools to deploy and manage Scalar DL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 3**. Please refer to the following document for more details.  
-
-* [Create a bastion server]() // TODO: Create a new document
+For executing some tools to deploy and manage ScalarDL Ledger on EKS, you need to prepare a bastion server in the same VPC of the EKS cluster you created in **Step 2**. See [Create a bastion server](./CreateBastionServer.md) for more details.
 
 ## Step 5. Prepare a custom values file of Helm 
```

Comment:
Do we need to configure two custom values files in this case (one for ScalarDL Ledger and one for ScalarDL Schema Loader (for Ledger)? If so, let's use the following:

```suggestion
## Step 5. Prepare custom values files for the Scalar Helm Charts for both ScalarDL Ledger and ScalarDL Schema Loader
```

If only one custom values file needs to be configured, let's use the following:

```suggestion
## Step 5. Prepare a custom values file for the Scalar Helm Chart
```